### DISCUSSION TITLE
Tooltip

### DISCUSSION THREAD

**Discussion by srednasoft** [img-mapper/img-mapper#70] [Q&A]
Hi,
Are there any good examples or tips on showing a tooltip when mouse enters a map area.
The tooltip should show up where the mouse cursor is.
I have not figured out how to do this

**Comment by NishargShah** [comment]
There are many issues related to this, please check the closed issues/discussion. Let me know if you still facing issues with tooltips, I will help you.

**Reply by srednasoft** [reply]
I have done searching but can’t find anything about tooltips?😳

**Reply by srednasoft** [reply]
I can do a Modal/Popup when there is mouse events on an area, but I have not been able to make a Tooltip connected to the area tag. All of the Tooltip plugins I have found will surround the Tooltip around the tag.
The html area tags don´t have id´s, so it´s not so easy to find the correct area tag with javascript either.
I would be very happy if you could help me to the right track with this problem.

**Reply by srednasoft** [reply]
Hi @NishargShah ,
any news on Tooltips?
I'm still strugling with this
